modules/spo-container-profile-instances.adoc

```diff
@@ -73,30 +82,40 @@ spec:
 +
 [source,terminal]
 ----
-$ oc delete deployment nginx-deploy
+$ oc delete deployment nginx-deploy -n my-namespace
 ----
 
 . To merge the profiles, delete the profile recording by running the following command:
 +
 [source,terminal]
 ----
-$ oc delete profilerecording test-recording
+$ oc delete profilerecording test-recording -n my-namespace
 ----
 
 . To start the merge operation and generate the results profile, run the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ oc get {object} -lspo.x-k8s.io/recording-id=test-recording
+$ oc get {object} -lspo.x-k8s.io/recording-id=test-recording -n my-namespace
 ----
+ifdef::spo-selinux[]
 +
-.Example output
+.Example output for {object}
 [source,terminal]
 ----
-NAME                          USAGE                                         STATE
-test-recording-nginx-record   test-recording-nginx-record_mytest1.process   Installed
+NAME                          USAGE                                          STATE
+test-recording-nginx-record   test-recording-nginx-record_mytest1.process    Installed
 ----
-
+endif::[]
+ifdef::spo-seccomp[]
++
+.Example output for {object}
+[source,terminal]
+----
+NAME                          STATUS       AGE
+test-recording-nginx-record   Installed    55s
+----
+endif::[]
```

Comment:
I might be missing something, but I can't see this in the preview?

Reply:
Dang! @abrennan89 good catch.

Reply:
It worked fine for selinuxprofile....arrg.

Reply:
This has been fixed!